### PR TITLE
July update: vaurca and mining fixes

### LINKED ISSUE #5036: Vaurca softsuits do not use their object sprites when retrofitted.
As it says in the title. The sprites for said objects exist, yet they are unused.

### LINKED ISSUE #5037: The Vaurca combat exoskeleton does not use its object sprite
Said sprite exists, and replacement should be trival.

### LINKED ISSUE #5046: If you hit a wall with say, a pickaxe, you'll start climbing it as well as hit it
Pls stop climbing from engaging if you hit the wall with an object.

### PR DESCRIPTION
**Pull request by Alberyk**
-fixes #5046
-fixes #5037
-fixes #5036
-changes that you can only climb walls by being in the grab intent, to avoid falls by accident when clicking a wall
-changes the cooldown of the warform at bygone's request

### CODE REVIEW

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/obj/clothing/species/vaurca/hats.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
rig0-mining||rig0-mining|Created
rig0-medical||rig0-medical|Created
rig0-atmos||rig0-atmos|Created
rig0-engineering||rig0-engineering|Created
rig0-syndie||rig0-syndie|Created
rig0-sec||rig0-sec|Created
</details>
<details><summary>icons/obj/clothing/species/vaurca/suits.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
rig-mining||rig-mining|Created
rig-medical||rig-medical|Created
rig-atmos||rig-atmos|Created
rig-engineering||rig-engineering|Created
rig-syndie||rig-syndie|Created
rig-sec||rig-sec|Created
</details>
<details><summary>icons/obj/rig_modules.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
vaurca_rig||vaurca_rig|Created
</details>

**Comment by BurgerLUA** [comment]
I think it would be smarter just to add a right clickable verb to all wall turfs.

**Comment by LordFowl** [comment]
Why would that be smarter at all? Right-clicking should be avoided when possible.

File: code/modules/clothing/spacesuits/void/void.dm [hunk 1]
```diff
@@ -20,7 +20,8 @@
 	sprite_sheets_obj = list(
 		"Unathi" = 'icons/obj/clothing/species/unathi/hats.dmi',
 		"Tajara" = 'icons/obj/clothing/species/tajaran/hats.dmi',
-		"Skrell" = 'icons/obj/clothing/species/skrell/hats.dmi'
+		"Skrell" = 'icons/obj/clothing/species/skrell/hats.dmi',
+		"Vaurca" = 'icons/obj/clothing/species/vaurca/hats.dmi',
```
Comment (skull132) [inline code/modules/clothing/spacesuits/void/void.dm]: You have set off the comma detector. This has a unnecessary comma at the end.

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 2]
```diff
@@ -580,7 +580,7 @@
 		to_chat(src, "<span class='danger'>You cannot charge in your current state!</span>")
 		return
 
-	last_special = world.time + 200
+	last_special = world.time + 10
```
Comment (LordFowl) [inline code/modules/mob/living/carbon/human/human_powers.dm]: Is this meant to be only + 10? Isn't trample their strongest ability?

Comment (Alberyk) [reply code/modules/mob/living/carbon/human/human_powers.dm]: As far I have been told, it was supossed to be used with the other powers, but I can increase the cooldown.

Comment (LordFowl) [reply code/modules/mob/living/carbon/human/human_powers.dm]: That would not work well anyways, due to the fact that all human powers share 1 cooldown.

**Comment by BurgerLUA** [comment]
Because as proven with it's current implementation, and likely it's future implementation, you're going to get a lot of accidental wall climbs.

**Pull request merged**